payloads/v10/user.ts

```diff
@@ -151,8 +151,12 @@ export enum UserFlags {
 	 * User's account has been quarantined based on recent activity
 	 *
 	 * @unstable This user flag is currently not documented by Discord but has a known value which we will try to keep up to date.
+	 *
+	 * @privateRemarks
+	 *
+	 * This value would be 1 << 44, but bit shifting above 1 << 30 requires bigints
```

Comment:
Can't we keep this on the same line just like with `@unstable`?

Reply:
According to tsdoc, it's a block tag (which means: no)
https://tsdoc.org/pages/tags/privateremarks/